app/admin.js

```diff
@@ -334,15 +430,234 @@ const handleVerification = co(function *(args) {
   }
 });
 
+const assertFileDoesNotExist = function(filename) {
+    try {
```

Comment:
there are `fs.exists()` and `fs.existsSync()` functions in node -- would make a lot more sense to use those